examples/history_vehicles_replacement_for_imitation_learning.py

```diff
@@ -20,6 +20,9 @@ def act(self, obs):
 
 def main(scenarios, headless, seed):
     scenarios_iterator = Scenario.scenario_variations(scenarios, [])
+    first_time = True
```

Comment:
If we initialize at the beginning with an empty agent_interfaces - `SMARTS(agent_interfaces={})`, the code would be more concise.